PyGame_Project/MVC/Controller/CLI_controller.py

```diff
@@ -188,23 +205,46 @@ def main_response(userInput):
 
 # when user wants to load a saved game
 def load_save_game():
-  ## all load options
-  print_load_options()
-  file_name = user_input(4)
+  # gets all of the save file names (removes the ".json" for nicer printing in command window)
```

Comment:
This should probably be in the view, it's a "complex" function with few looping prints.